interfaces/builtin/browser_support.go

```diff
@@ -72,6 +72,9 @@ owner /{dev,run}/shm/.io.nwjs.* mrw,
 /run/user/[0-9]*/snap.@{SNAP_INSTANCE_NAME}/{,.}com.google.Chrome.*/SS r,
 /run/user/[0-9]*/snap.@{SNAP_INSTANCE_NAME}/{,.}com.microsoft.Edge.*/SS r,
 
+# Allow access to Jupyter notebooks (LP: #1959417)
+owner @{HOME}/.local/share/jupyter/** rw,
```

Comment:
I would suggest to add a comment here saying that this access is temporary, and only granted until [bug 1959417](https://bugs.launchpad.net/ubuntu/+source/jupyter-notebook/+bug/1959417) is fixed upstream.

At the same time we should start working with upstream to either:
1. Change the path where notebooks are generated, or
2. Change the text displayed to not invite the user to open that path with the browser

Reply:
I'm trying to get in touch with the jupiter-notebook developers here: https://github.com/jupyter/notebook/discussions/6436